operations/services.md

```diff
@@ -97,10 +97,13 @@ Returns all services offered by the enterprise.
 | `EndOffset` | string | required | Offset from the end of the time unit defining the default end of the service orders in ISO 8601 duration format. |
 | `OccupancyStartOffset` | string | required | Offset from the start of the time unit defining the occupancy start of the service in ISO 8601 duration format that is considered regarding the availability and reporting. |
 | `OccupancyEndOffset` | string | required | Offset from the end of the time unit defining the occupancy end of the service in ISO 8601 duration format that is considered regarding the availability and reporting. |
-| `TimeUnit` | [Time unit](#time-unit) | required | Time unit of the service. |
+| `TimeUnit` | [Time unit period](#time-unit-period) | required | Time unit period of the service. |
+
+
+#### Time unit start and end
+Time units represent a fixed, finite time interval: a minute, a day, a month, etc. A Time unit defines the operable periods for a bookable service.
```

Comment:
This seems to be a duplicity with definition of TimeUnit
There is 
Time unit
* Day,
* Month 
The description should be there no?